### PR TITLE
[ID] Add "Rules/Explicit content" and an example screenshot

### PR DESCRIPTION
**Pull request by Nivalyx**
Indonesian translation for the Explicit content (http://osu.ppy.sh/wiki/Rules/Explicit_Content) wiki article.

((also added a certain image to both the [EN] and [ID] pages to provide a visual example that goes along with the article))

### CODE REVIEW

File: wiki/Rules/Explicit_Content/id.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Konten eksplisit
+
+![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap dengan ditandai dengan label 'eksplisit'.")
```
Comment (wowcake) [inline wiki/Rules/Explicit_Content/id.md:3]:
```suggestion
![Tampilan beatmap yang mengandung konten eksplisit](img/explicit-tag.jpg "Contoh beatmap yang ditandai dengan label 'eksplisit'.")
```

Comment (wowcake) [reply wiki/Rules/Explicit_Content/id.md:3]: it doesn't feel natural using ```dengan``` when the next word after it is going to use ```dengan``` aswell.

**Review by wowcake — CHANGES_REQUESTED** [review]

**Comment by wowcake** [comment]
will check again later

File: wiki/Rules/Explicit_Content/id.md (hunk 1 above)
Comment (Nivalyx) [reply wiki/Rules/Explicit_Content/id.md:3]: ya this is an oversight lol \:P

tyty

File: wiki/Rules/Explicit_Content/id.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Ketika membuat beatmap, para mapper dapat menyematkan label 'eksplisit' pada beatmap-beatmap buatan mereka di situs web osu! untuk menandai bahwa beatmap-beatmap yang bersangkutan mengandung konten eksplisit.
+
+Artikel ini menjelaskan seputar berbagai hal yang terkait dengan dengan label tersebut beserta tata cara penggunaannya.
+
+## Hal-hal apa saja yang termasuk ke dalam konten eksplisit?
+
+osu! mengizinkan pemakaian **konten audio** yang bersifat eksplisit (baik dari segi *tema* ataupun *lirik* lagu) hingga batasan tertentu.
+
+**Di sisi lain, osu! tidak membenarkan penggunaan gambar dan video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
```
Comment (mousesalt) [inline wiki/Rules/Explicit_Content/id.md:13]:
```suggestion
**Di sisi lain, osu! tidak membenarkan penggunaan gambar maupun video yang bersifat senonoh - seluruh elemen visual yang ada tetap harus tunduk pada [Pertimbangan Konten Visual](/wiki/Rules/Visual_Content_Considerations).**
```

Comment (mousesalt) [reply wiki/Rules/Explicit_Content/id.md:13]: this because `maupun` could be related such `dan/atau`.

File: wiki/Rules/Explicit_Content/id.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Pengecualian di sini hanya diberikan pada elemen-elemen visual senonoh yang mengandung unsur estetika seni (*artistic use*) dalam nuansa dan konteks pemakaiannya.
+
+Pada intinya, selama dilabeli sebagaimana mestinya, lagu-lagu yang bersifat eksplisit dapat dipergunakan dan disebarluaskan dalam bentuk beatmap di dalam lingkungan osu!.
+
+## Lirik lagu yang seperti apa yang dikategorikan eksplisit?
+
+Untuk dapat dikategorikan eksplisit, suatu lagu harus memiliki lirik yang mengandung hujatan, cacian, atau kata-kata kasar lainnya dalam porsi yang signifikan. Apabila kata-kata kasar tersebut hanya muncul sekali atau dua kali dalam keseluruhan lirik lagu yang ada, maka lagu yang bersangkutan tidak perlu untuk dilabeli eksplisit selama tema lagu yang dimaksud berada pada ranah PG-13 atau lebih rendah. Apabila kamu tidak yakin apakah suatu lagu terbilang eksplisit atau tidak, sebaiknya kamu mengasumsikan bahwa lagu tersebut adalah lagu yang eksplisit demi kemaslahatan bersama.
```
Comment (mousesalt) [inline wiki/Rules/Explicit_Content/id.md:21]:
```suggestion
Lirik lagu yang dapat dikategorikan eksplisit; memiliki lirik yang mengandung hujatan, cacian, atau kata-kata kasar lainnya dalam porsi yang signifikan. Apabila kata-kata kasar tersebut hanya muncul sekali atau dua kali dalam keseluruhan lirik lagu yang ada, maka lagu yang bersangkutan tidak perlu untuk dilabeli eksplisit selama tema lagu yang dimaksud berada pada ranah PG-13 atau lebih rendah. Apabila kamu tidak yakin apakah suatu lagu terbilang eksplisit atau tidak, sebaiknya kamu mengasumsikan bahwa lagu tersebut adalah lagu yang eksplisit demi kemaslahatan bersama.
```

Comment (mousesalt) [reply wiki/Rules/Explicit_Content/id.md:21]: the use of `harus` seems to mean force to choose the songs that have explicit lyrics to get mapped.

File: wiki/Rules/Explicit_Content/id.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Singkatnya, suatu lagu harus memiliki lirik yang mengandung kata-kata kasar yang diucap berulang-ulang untuk dapat dikategorikan eksplisit.
```
Comment (mousesalt) [inline wiki/Rules/Explicit_Content/id.md:23]:
```suggestion
Singkatnya, suatu lagu yang memiliki lirik mengandung kata-kata kasar diucapkan berulang-ulang untuk dapat dikategorikan eksplisit.
```

Comment (mousesalt) [reply wiki/Rules/Explicit_Content/id.md:23]: the case similar to above, reduce the use of `yang`.

File: wiki/Rules/Explicit_Content/id.md (hunk 2 above)
Comment (Nivalyx) [reply wiki/Rules/Explicit_Content/id.md:13]: kk applied

File: wiki/Rules/Explicit_Content/id.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Rules/Explicit_Content/id.md:21]: yeah that's what the "Untuk dapat dikategorikan eksplisit" words are for ;~;

`Untuk dapat dikategorikan eksplisit, suatu lagu harus memiliki lirik yang mengandung hujatan...` --> `To be categorized as explicit, the song's lyrics has to contain profanity...`

File: wiki/Rules/Explicit_Content/id.md (hunk 4 above)
Comment (Nivalyx) [reply wiki/Rules/Explicit_Content/id.md:23]: did sth else here by simplifying the entire sentece

**Review by mousesalt — APPROVED** [review]
rechecked!

**Comment by wowcake** [comment]
checked 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**